src/uu/wc/src/count_fast.rs

```diff
@@ -134,3 +134,26 @@ pub(crate) fn count_bytes_and_lines_fast<R: Read>(
         }
     }
 }
+
+pub(crate) fn count_chars_fast<R: Read>(handle: &mut R) -> (WordCount, Option<io::Error>) {
```

Comment:
could you please comment this function profile? thanks
(with the rustdoc format)

Reply:
Done.